pbs/usersync.go

```diff
@@ -325,6 +367,15 @@ func (deps *UserSyncDeps) OptOut(w http.ResponseWriter, r *http.Request, _ httpr
 	}
 }
 
+// getExpiry gets an expiry date for the cookie, assuming it was generated right now.
+func getExpiry(familyName string) time.Time {
+	ttl := 7 * 24 * time.Hour
```

Comment:
let's add this to the viper config.

Reply:
I looked into this... and while it's possible, I'm not sure I want to go down this route in this PR. The question of "how do we pass highly-visible data throughout the project?" is an architectural one... and I can think of a few very different ways of doing it.

Right now it's only used by `pbs_light`, and the strategy seems to be: "add a `*Configuration` argument to any functions which need it". I'm not a huge fan because there are other global objects too (e.g. Metrics), and it makes testing difficult. UserSync stuff might only care about ~5 config properties, but if the argument is `*Configuration`, I have to make a _whole_ Configuration if I want to write black-box tests for the module.

I think I have a mental vision for how we can handle all this cleanly... but the code changes are pretty substantial, and (I hope) there'll be a fair amount of back and forth on it.

Let me know if you still think configurability is essential here. Otherwise let's open a new ticket for establishing a pattern for handling dependencies, and I'll take a shot at it next sprint.

Reply:
+1 to smaller PRs (i.e. avoid the re-architecture). The middle ground is simply to make it a const up with the `customBidderTTLs` so that all of the configurations are approximately in the same space, instead of buried in the code. Having them hard coded there should be fine for now given that this should rarely change.

Reply:
I'd agree with that.